packages/react/slider/src/Slider.tsx

```diff
@@ -524,14 +545,13 @@ SliderTrack.displayName = TRACK_NAME;
 const RANGE_NAME = 'SliderRange';
 const RANGE_DEFAULT_TAG = 'span';
 
-type SliderRangeOwnProps = {
-  children: never;
-};
-
-const SliderRange = forwardRefWithAs<
+type SliderRangeOwnProps = Merge<Polymorphic.OwnProps<typeof Primitive>, { children?: never }>;
```

Comment:
So yeah again related to my previous question, we do `never` here, but won't it get re-added anyway via the DOM props if we don't remove the never keys ourselves in our types internally?